projects/hathor-wallet-headless/0001-multisig-create-mint-melt-token.md

```diff
@@ -0,0 +1,319 @@
+* `$.completeSignatures` -- that represents the completeness of signatures required to use the transaction
+* `$.tx.inputs[*].signed` -- that indicates this input has a signature
+* `$.tx.outputs[*].mine` -- that indicates this output belongs to this wallet
+* `$.balance[*].tokens` -- that contains the balance for the token
+* `$.balance[*].authorities` -- that contains the balance of mint and melt for the token
+
+## Initialize read-only MultiSig wallet
+
+This wallet is used by the coordinator service to sign or sign-and-push the transaction given the signatures collected by the service.
+
+A read-only wallet only needs the `xpubkey` to initialize. The MultiSig wallet can be initialized with `seedKey` or `multisigKey`, but initializing with `multisigKey` there is no need to configure the `seeds` property in the configuration.
+
+By combining the two initializations we have a MultiSig read-only wallet, as we can see bellow:
+```bash
+curl -X POST --data "wallet-id=my1" \
+	--data "xpubkey=xpub...FHB" \
+	--data "multisigKey=mymultisigwallet" \
+	--data "multisig=true" \
+	http://localhost:8000/start
+```
+
+In the presented configuration the `xpubkey` doesn't have any role in the formation of `multisigData`, it servers only to avoid the requirement to configure the `seeds` for the MultiSig wallet.
+
+With this configuration one can request the creation of transaction proposals but can't call `get-my-signatures` with success. However, after collect all the signatures it is possible to call `sign` or `sign-and-push`.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Despite the current design provides a minimal set of operations to enable the EVM compatible bridge, it still lacks some operations like "create authority", "destroy authority", and "delegate authority", that could equip an organization to take full control over its assets in a MultiSig wallet. Also, the token creation don't supports NFT token.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- The addition of specialized commands is better than change the behavior of an existing command because this approach avoids produce breaking changes.
+
+# Prior art
+[prior-art]: #prior-art
+
+* This design is a complement for the  [EVM compatible bridge design](https://github.com/HathorNetwork/rfcs/blob/doc/evm-compatible-brigde/projects/evm-compatible-bridge/design.md#federation-service). 
+* The commands to create token, mint and melt includes a fine tuned authority creation after the PRs [#291](https://github.com/HathorNetwork/hathor-wallet-headless/pull/291) and [#293](https://github.com/HathorNetwork/hathor-wallet-headless/pull/293) in the wallet-headless.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+* Should NFT data be allowed in the create token endpoint for the MultiSig wallet, or do it deserves its own endpoint, as it happens for the single wallet?
```

Comment:
How would that be helpful? Is there a use case in sight for this?

Reply:
Hmm... The use case I have in sight is this: an organization that needs to mint a new NFT collection but use a multisig wallet. I far as I know, we don't have a partner that need it, but this is a possibility.